src/main/java/com/treemoval/data/Forest.java

```diff
@@ -22,6 +21,7 @@
 public class Forest {
 
     List<Tree> trees = new ArrayList<>();
+    private double safeDistance = 5; //need to implement way for user to modify
```

Comment:
In IntelliJ, if you put `// todo ...blah blah` (for example `// todo need to implement a way for the user to modify this`), the IDE will mark the text yellow, and it will be tagged alongside your scroll bar. Pretty useful stuff, especially since we're all using IntelliJ.

Reply:
noted, and added